widget/accordion_internal_test.go

```diff
@@ -13,7 +13,7 @@ import (
 func TestAccordionContainer_Toggle(t *testing.T) {
 	ai := NewAccordionItem("foo", NewLabel("foobar"))
 	ac := NewAccordionContainer(ai)
-	acr := test.WidgetRenderer(ac).(*accordionContainerRenderer)
+	acr := test.WidgetRenderer(ac).(*accordionRenderer)
```

Comment:
nit: the variable was named as an abbreviation of the type
```suggestion
	ar := test.WidgetRenderer(a).(*accordionRenderer)
```